### PR TITLE
UI/check widget visibility

### PR DESCRIPTION
**Pull request by NancyMcCallB**
Allow apps to query the frontstage to find our if a specific widget is currently visible.
Return WidgetState.Floating for widgets in a floating state.

### CODE REVIEW

File: common/api/core-frontend.api.md [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ export function areaToEyeHeightFromGcs(view3d: ViewState3d, area: GlobalLocation
 
 // @internal
 export interface AttachToViewportArgs {
-    drawingToSheetTransform?: Transform;
+    // (undocumented)
```
Comment (pmconne) [inline common/api/core-frontend.api.md:1254]: What happened here? This is documented on master.

Comment (pmconne) [reply common/api/core-frontend.api.md:1254]: Actually, almost none of these api.md file changes should be present.

**Review by bsteinbk — APPROVED** [review]

File: common/api/core-frontend.api.md (hunk 1 above)
Comment (bsteinbk) [reply common/api/core-frontend.api.md]: I pulled the branch and re-ran extract-api - seems to be OK now.

Comment (NancyMcCallB) [reply common/api/core-frontend.api.md]: It's some shenanigans when I merged in from master and then did an extract-api. If I can't fix it on this branch, I'll just start over.

**Pull request merged**